### PR TITLE
fix: title in Kafka example

### PR DESCRIPTION
**Pull request by fmvilas**
Yeah, I don't think it needs explanation 😄

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -113,7 +113,7 @@
     "markdown-toc": "^1.2.0",
     "postcss": "^8.2.13",
     "raw-loader": "^4.0.2",
-    "react-scripts": "5.0.1",
+    "react-scripts": "4.0.3",
```
Comment (magicmatatjahu) [inline package.json:116]: Fixes https://github.com/asyncapi/studio/pull/451

**Comment by fmvilas** [comment]
@magicmatatjahu why are you including all these commits in this PR? 🤔

**Comment by magicmatatjahu** [comment]
@fmvilas

- we forgot to update version `2.4.0` -> `2.5.0`. It's fix so we can do it in one PR
- dependantbot updated `react-scripts` to the `^5` version and it breaks the build, so I revert to the `^4` version

**Review by magicmatatjahu — APPROVED** [review]

**Pull request merged**